src/app/components/search-bar/search-bar.component.html

```diff
@@ -1,12 +1,10 @@
-<div id="search-bar">
-  <input
-    #searchBar
-    type="text"
-    placeholder="Search"
-    id="searchBar"
-    (keyup.enter)="handleSearch(searchBar.value)"
-  />
-  <button
-    type="button" (click)="handleSearch(searchBar.value)"> Search
-  </button>
-</div>
+<div class="search-pf has-button Search">
```

Comment:
This file has indentation with 4 spaces. It should be 2.

Reply:
Done